### PR TITLE
remove an obsolete monkeypatch

### PR DESCRIPTION
**Pull request by andre-merzky**

### CODE REVIEW

**Comment by andre-merzky** [comment]
I am not sure why the tests fail here - they seem to work alright alright.  Any insight?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -43,7 +43,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ '3.7', '3.8', '3.9', '3.10', '3.11' ]
+        python-version: [ '3.7', '3.8', '3.9', '3.10', '3.11', '3.12',
+                          '3.13.0-rc.1' ]
```
Comment (mtitov) [inline .github/workflows/ci.yml:47]: seems that rc2 is available

Comment (mtitov) [reply .github/workflows/ci.yml:47]: on another note - should we skip 3.13 until coverage is fixed there?
OR we can have a draft PR with 3.13, and check it every sprint until it is passed

**Review by mtitov — APPROVED** [review]
LGTM!

**Pull request merged**